docs/source/onnxruntime/usage_guides/amdgpu.mdx

```diff
@@ -0,0 +1,117 @@
+# Accelerated inference on AMD GPUs
+
+By default, ONNX Runtime runs inference on CPU devices. However, it is possible to place supported operations on an AMD Instinct GPU, while leaving any unsupported ones on CPU. In most cases, this allows costly operations to be placed on GPU and significantly accelerate inference.
+
+This guide will show you how to run inference on the `ROCMExecutionProvider` execution provider that ONNX Runtime supports for AMD GPUs:
+
+## Installation
+
+#### 1. ROCM Installation (V 5.7.X)
+To install ROCM, please follow the [ROCM installation guide](https://rocm.docs.amd.com/en/latest/deploy/linux/index.html).
+
+#### 2. Torch Installation with ROCM Support
+
+```bash
+pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/rocm5.7
+```
+
+> **Note1:** Exporting with SDPA may encounter issues with torch versions below 2.1.0. To ensure smooth operation, consider upgrading to torch 2.1.0 or newer.
```

Comment:
```suggestion
```

The suggestion is PyTorch nightly in the above. Also fixed in https://github.com/huggingface/optimum/pull/1547